src/www/ui/async/AjaxShowJobs.php

```diff
@@ -159,7 +159,7 @@ protected function getGeekyScanDetailsForJob($job_pk)
           }
           if (filesize($row[$field]) > self::MAX_LOG_OUTPUT) {
             $value = "<pre>" .file_get_contents($row[$field],false,null,-1,self::MAX_LOG_OUTPUT)."</pre>"
-                    .'<a href="'.Traceback_uri() . '?mod=download&log=' . $row['jq_pk'] . '">...</a>';
+                    .'<a href="'.Traceback_uri() . '?mod=download&log=' . $row['jq_pk'] . '">display full log</a>';
```

Comment:
```suggestion
                    .'<a href="'.Traceback_uri() . '?mod=download&log=' . $row['jq_pk'] . '">Download full log</a>';
```

Reply:
Since the link provides a file to download, the display should be replaced with download in the message.